### ISSUE TITLE
v1.16 release sha256sum hashes incorrect

### ISSUE THREAD

**Issue by frots** [GoogleCloudPlatform/cloudsql-proxy#322]
Hello,

The published sha256sum hashes for v1.16 on https://github.com/GoogleCloudPlatform/cloudsql-proxy/releases (@kurtisvg) do not match when I hash the binaries. 
6e4df1e2b62e41b3c01818f75dd46a99cb0e9d385c3b6237281251e28cb38432  Downloads/cloud_sql_proxy.linux.amd64
9d543e4ac1df190d2f54399a7ff46f0b38d0f0a72a0fc69f6579e24ca50cdea9  Downloads/cloud_sql_proxy.linux.386

Can you please confirm the binaries are unaltered and safe to use?

Thanks,

**Comment by kurtisvg** [comment]
Hey frots@ - thanks for bringing this to our attention. 

We've verified that the hashes you've gotten are in fact correct, and match out records. We've updated the sha's on the releases page to their correct versions. 

For the sake of transparency, what happened here was we use a script to trigger our releases and push the binaries to different locations. During this last release some updates to this script we made had failed, and we ran it 2-3 times while debugging. Currently, I manually copy the sha's generated as part of this script onto our releases page. However in this case, I copied the sha's from a different build than the one we published. 

This build had a different sha because a dependency changed. We are working both to 1) Automated the releases page so I can't mess this up in the future, and 2) Do a better job of pinning our dependencies so that each tag of the build is completely reproducible.
